install/install-kritis.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# Command line Args Default
+NAMESPACE="default"
+GAC_SECRET="gac-secret"
```

Comment:
If we're assuming the default secret name is gac-secret, we should change it in [values.yaml](https://github.com/grafeas/kritis/blob/master/kritis-charts/values.yaml#L49) and in the README, where we create secrets.

Reply:
yeah we can do that. Will make those changes.